src/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Button.css';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    const { value } = this.props;
```

Comment:
`name` seems like a better description for the button's name.
```suggestion
    const { name } = this.props;
```